CONTRIBUTING.md

```diff
@@ -1,14 +1,12 @@
-# Developing cadence-client
+# Developing temporal-client
 
-This doc is intended for contributors to `cadence-client` (hopefully that's you!)
+This doc is intended for contributors to `temporal-client` (hopefully that's you!)
 
 **Note:** All contributors also need to fill out the [Uber Contributor License Agreement](http://t.uber.com/cla) before we can merge in any of your changes
```

Comment:
Remove or add reference to Temporal one.

Reply:
Will do this in next change